test/e2e/data/infrastructure-docker/main/cluster-template-upgrades-runtimesdk/cluster-runtimesdk.yaml

```diff
@@ -24,6 +24,8 @@ spec:
         - class: "default-worker"
           name: "md-0"
           nodeDeletionTimeout: "30s"
+          nodeVolumeDetachTimeout: "5m"
```

Comment:
q: are we using volumes in this test?

Reply:
I'm not sure, just noticed that there was an inconsistency with how the template was updated, but it shouldn't actually affect the outcome of this test. I'll try and circle back on the actual source of the flake.